cmd/snap/cmd_snapshot.go

```diff
@@ -410,9 +417,22 @@ func init() {
 				desc: i18n.G("The filename of the export"),
 			},
 		})
-	// This command is hidden because there's no corresponding
-	// "import-snapshot" to consume the produced data.
-	// TODO: implement import-snapshot and remove the hidden attribute.
+	// XXX: this command is hidden because import/export is not complete
+	cmd.hidden = true
```

Comment:
This could be removed, no?

Reply:
Maybe, but I think it's better to wait for #9036 to actually remove it. This PR adds the scaffolding for the import but the actual backend command for import will just return an error so it seems best to not show it just yet until it fully works. WDYT?

Reply:
Right, makes sense, we should unhide it in the last PR, thanks.